YoutubeDownloader/ViewModels/Dialogs/DownloadMultipleSetupViewModel.cs

```diff
@@ -36,19 +40,20 @@ SettingsService settingsService
     public VideoQualityPreference SelectedVideoQualityPreference { get; set; } =
         VideoQualityPreference.Highest;
 
-    public void OnViewLoaded()
+    protected override void OnViewLoaded()
     {
         SelectedContainer = settingsService.LastContainer;
         SelectedVideoQualityPreference = settingsService.LastVideoQualityPreference;
+        SelectedVideos.CollectionChanged += (sender, args) => OnPropertyChanged(nameof(CanConfirm));
```

Comment:
Can't use that `WhenAnyValue` extension method here?

Reply:
WhenAnyValue only works with PropertyChangedEvents, there is an ObservableCollection replacement that one can subcribe to like with WhenAnyValue called DynamicData which is really powerful but also really complex (https://www.reactiveui.net/docs/handbook/collections.html)